utils/land-patterns.stanza

```diff
@@ -7,8 +7,12 @@ defpackage ocdb/land-patterns :
   import jitx/fonts
   import jitx/commands
 
-; ====== Reference Label Helpers ===============================================
+; ====== Argument constraint Helpers ===========================================
+defn ensure-even-positive! (value: Int, name: String):
+  if value < 2 or value % 2 == 1:
+    fatal("%_ only supports a positive even number of pins." % [name])
```

Comment:
What's your methodology for deciding which generators require this check? Based on tests errors?

Reply:
I googled sop65 and soic127p to check. But yes they crash for odd integers